### ISSUE TITLE
ENHANCEMENT - FT 5.7 + SERVER - Appended PDF standard reports

### ISSUE THREAD

**Issue by atl2017** [smap-consulting/smapserver#377]
### Expected Behaviour. Now that we have implemented #213  a button that trigger a form, we should append standard pdf reports as just one with main (source) and appended (target).

Example like
[TriggerFormsPDFProposal.pdf](https://github.com/smap-consulting/smapserver/files/2516913/TriggerFormsPDFProposal.pdf)

**Comment by nap2000** [comment]
The system is evolving to use a lot of form linking.  

Previously you had a main form and then child forms specified as "begin repeats". So those sort of links are well handled.  You can request an XLS export of the data and the forms will be joined to present a single view of al the data in the survey.

What is happening now is that you can specify a lookup of a key in form B from form A.  These forms having been specified in completely different surveys.  Once you store B's key in the record for A then the data is linked.  However it is weak linking and the reporting / analytics doesn't really handle it at the moment.  Using the form launcher is just a variant on this which allows you to create the record you are linking to at the same time as creating the record you are linking from.

I think the main focus for adding analytics to this capability will initially be to support it in BI tools like tableau and Power BI.  So I'm going to aim to implement support via the new odata interface first.   Creating linked PDF reports would be a lower priority.

**Comment by atl2017** [comment]
Is it possible to include the link equivalent to the action that trigger button event as below?

![image](https://user-images.githubusercontent.com/29519901/47534882-97db2300-d87e-11e8-80e4-0ca6890bb5b8.png)

**Comment by nap2000** [comment]
Yes its possible.  Like a hierarchical list of begin repeat forms is shown when specifying a report we could show a hierarchical list of weakly linked forms.

**Comment by atl2017** [comment]
Great if we can include this. There will be many uses of this due that Security Companies uses some "Digital Minutes"

**Comment by atl2017** [comment]
In the example below "triggerforms" trigger form2, execute it and received back key, ie 1:

![image](https://user-images.githubusercontent.com/29519901/49792214-278a4300-fd00-11e8-808a-193be91398bb.png)

So what I need is a way from "triggerforms" where we can have access to link to form pdf.  Maybe it could be by including a new option in parameters like:  form_identifier=s411_9786; key_question=q2; return=pdf_link

return could have the following options: 

-  uuid (as current does)
-  key (as I proposed before to return )
-  pdf_link (link to generate pdf equivalent to click in "Download PDF" see image below)

![image](https://user-images.githubusercontent.com/29519901/49792739-74bae480-fd01-11e8-9f05-360e0c543b6a.png)

So when "triggerform" pdf is generated and return=pdf_link it could be clickable also in Analysis table

![image](https://user-images.githubusercontent.com/29519901/49792947-f579e080-fd01-11e8-9ca8-75eb0e531098.png)

![image](https://user-images.githubusercontent.com/29519901/49793036-2ce88d00-fd02-11e8-8de4-9a100f25c42f.png)
